pom.xml

```diff
@@ -291,6 +291,17 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
```

Comment:
This would be removed before committing right - this is just to test 1.12 before release?

Reply:
Yes.